patches/enable-media-replace.0001.fix.disable-search-replace-method.patch

```diff
@@ -0,0 +1,38 @@
+From 1168a718bcd97e89125c9ade42c8715578154b04 Mon Sep 17 00:00:00 2001
+From: Nabi <artsanaat@gmail.com>
+Date: Mon, 24 Jul 2023 17:45:05 +0330
+Subject: [PATCH] Added a parameter to disable Search and Replace in Enable
+ Media Replace plugin.
+
+---
+ classes/replacer.php | 7 ++++---
+ 1 file changed, 4 insertions(+), 3 deletions(-)
+
+diff --git a/classes/replacer.php b/classes/replacer.php
+index cc3fd4b10..7f5680cf0 100644
+--- a/classes/replacer.php
++++ b/classes/replacer.php
+@@ -107,7 +107,7 @@ class Replacer
+   * @param $fileName String The fileName of the uploaded file. This will be used if sourcefile is not to be overwritten.
+   * @throws RunTimeException  Can throw exception if something went wrong with the files.
+   */
+-  public function replaceWith($file, $fileName)
++  public function replaceWith($file, $fileName, $doSearchReplace = true)
+   {
+       global $wpdb;
+       $this->targetName = $fileName;
+@@ -251,8 +251,9 @@ class Replacer
+       );
+ 
+       // Search Replace will also update thumbnails.
+-      $this->doSearchReplace($args);
+-
```

Comment:
I would recommend to leave the blank line after the call for diff context and reducing the size of the diff (increasing the chance that the patch will apply in the future)